src/Common/CliFolderPathCalculator.cs

```diff
@@ -58,19 +53,28 @@ public static string DotnetHomePath
                     home = Environment.GetEnvironmentVariable(PlatformHomeVariableName);
                     if (string.IsNullOrEmpty(home))
                     {
-                        throw new ConfigurationException(
-                                string.Format(
-                                    LocalizableStrings.FailedToDetermineUserHomeDirectory,
-                                    DotnetHomeVariableName))
-                            .DisplayAsError();
+                        throw new Exception(); // TODO
+                        // throw new ConfigurationException(
+                        //         string.Format(
+                        //             LocalizableStrings.FailedToDetermineUserHomeDirectory,
+                        //             DotnetHomeVariableName))
+                        //     .DisplayAsError();
```

Comment:
Some work is needed here to be able to put this in `Common`. Moving the `ConfigurationException` and `DisplayAsError` is doable. I don't know how to deal with the `LocalizableStrings`.